keps/976-plain-pods/README.md

```diff
@@ -66,14 +68,11 @@ use of this API to implement queuing semantics for Pods.
 - Support queueing of individual Pods.
 - Support queueing of groups of Pods of fixed size, identified by a common label.
 - Opt-in or opt-out Pods from specific namespaces from queueing.
+- Support for [dynamic reclaiming of quota](https://kueue.sigs.k8s.io/docs/concepts/workload/#dynamic-reclaim)
+  for succeeded Pods.
 
 ### Non-Goals
 
-- Support for [dynamic reclaiming quota](https://github.com/kubernetes-sigs/kueue/issues/78)
-
-  This feature is incompatible with supporting Pod replacements without knowing the behavior of a
-  parent controller for the Pods.
-
```

Comment:
As I remember, we decided not to support these features to recommend users using Job or CustomJob.

https://github.com/kubernetes-sigs/kueue/pull/1031#discussion_r1291649359

Why did you decide to support these features on the naked pods?

Reply:
It doesn't add much complexity. We can just count the number of Succeeded Pods. We still will keep all of the finished Pods with finalizers.
On the other hand, it adds a lot of value to users to keep their clusters with high utilization.

Reply:
I don't have strong objections to supporting dynamic reclaiming quota on the plain pods.
I wanted to clarify the reason why expanding the plain pods integration.

Thanks.